src/flag_gems/ops/randn.py

```diff
@@ -12,24 +12,68 @@
 )
 from flag_gems.utils.shape_utils import volume
 
-try:
-    pair_uniform_to_normal = tl.pair_uniform_to_normal
-except AttributeError:
```

Comment:
Why these change?

Is `pair_uniform_to_normal` the same thing as `pair_uniform_to_normal_fast`?